examples/grpc/common/src/main/java/io/helidon/grpc/examples/common/StringClientApi.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.grpc.examples.common;
+
+import io.grpc.Channel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.stub.StreamObserver;
+import io.helidon.grpc.client.ClientServiceDescriptor;
+import io.helidon.grpc.client.GrpcServiceClient;
+import io.helidon.grpc.examples.common.Strings.StringMessage;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Spliterator;
+import java.util.Spliterators;
+
+/**
+ * A client to the {@link StringService} implemented with Helidon gRPC client API.
+ *
+ * @author Bin Chen
+ */
+public class StringClientApi {
+    private static String inputStr = "Test_String_for_Lower_and_Upper";
+    private static StringMessage inputMsg = StringMessage.newBuilder().setText(inputStr).build();
+
+    private StringClientApi() { }
+
+    /**
+     * Program entry point.
+     *
+     * @param args  the program arguments
+     *
+     * @throws Exception if an error occurs
+     */
+    public static void main(String[] args) throws Exception {
```

Comment:
Again, I know that you have copied this from another example but all of these `sleep` calls are unnecessary
and it is not a well thought out example. I have changed the `StringClient` that you have copied this from in the `grpc-mp` branch to get rid of them. You could do the same in this class.